packages/edit-post/src/index.js

```diff
@@ -155,3 +155,4 @@ export { default as PluginPostStatusInfo } from './components/sidebar/plugin-pos
 export { default as PluginPrePublishPanel } from './components/sidebar/plugin-pre-publish-panel';
 export { default as PluginSidebar } from './components/sidebar/plugin-sidebar';
 export { default as PluginSidebarMoreMenuItem } from './components/header/plugin-sidebar-more-menu-item';
+export { default as FullscreenMode } from './components/fullscreen-mode';
```

Comment:
We were talking about this recently in another PR. I believe we shouldn't export this from `edit-post`. 
We were considering creating a `@wordpress/admin-screen` package for reusable stuff across WP Admin screens.
it can include "Page/EditorSkeleton", "FullscreenMode", Potentially sidebar plugins...

Reply:
> We were considering creating a @wordpress/admin-screen package for reusable stuff across WP Admin screens.

Cool, that definitely makes more sense and it was just the thing I was looking for. Is there an open issue/pr for it that I can track and update this when it's ready?

Reply:
There's none. @jorgefilipecosta might be working on a PR and you can also add the package as part of this PR 🤷‍♂

Reply:
We have a PR with admin screen available at https://github.com/WordPress/gutenberg/pull/20698.

Reply:
While we figure out the best naming...  for this package, I think we can land this as an experimental component  in `block-editor` for now (like the skeleton)

Reply:
> While we figure out the best naming... for this package, I think we can land this as an experimental component in block-editor for now (like the skeleton)

@youknowriad I moved this component to `block-editor` package as suggested. So far its activation state has been persisted in `edit-post` store. I factored that out from the component itself so we can have a separate state in case of `edit-site`.